lib/ansible/module_utils/k8s/raw.py

```diff
@@ -91,24 +99,29 @@ def execute_module(self):
             }
         })
 
-    def perform_action(self, resource, definition):
+    def perform_action(self, resource, definition, facts=False):
         result = {'changed': False, 'result': {}}
         state = self.params.pop('state', None)
         force = self.params.pop('force', False)
         name = definition.get('metadata', {}).get('name')
         namespace = definition.get('metadata', {}).get('namespace')
+        labels = ','.join(['%s=%s' % (k, v) for (k, v) in definition.get('metadata', {}).get('labels', {}).items()])
```

Comment:
Kubernetes also supports `!=` and set based label selectors. It would probably be easier to have the user specify a selector and pass it through